cmd/siva/pack.go

```diff
@@ -135,6 +135,7 @@ func (c *CmdPack) writeFileHeader(fullpath string, fi os.FileInfo) error {
 
 func (c *CmdPack) writeFile(fullpath string, fi os.FileInfo) error {
 	f, err := os.Open(fullpath)
+	defer f.Close()
```

Comment:
this line should go after the error check, otherwise it will panic if `f` is `nil`

Reply:
actually, I'm going to decline this PR, the version that i was using/installed was from gopkg.in which fails for me, the close is here already. Would it be possible to update the version on gopkg.in ?